tests/test_generate_hooks.py

```diff
@@ -57,6 +58,19 @@ def test_ignore_hooks_dirs():
 
 
 @pytest.mark.usefixtures('clean_system', 'remove_additional_folders')
+def test_imports_in_hooks():
+    try:
+        generate.generate_files(
+            context={
+                'cookiecutter': {'pyhooks': 'pyhooks'}
+            },
+            repo_dir='tests/test-hook-imports/',
+            output_dir='tests/test-hook-imports/')
+    except FailedHookException:
+        pytest.fail("Unexpected FailedHookException...")
```

Comment:
Test it does not raise an ** FailedHookException** when we import  something from hooks